server/src/main/java/com/objectcomputing/checkins/services/action_item/ActionItemRepository.java

```diff
@@ -1,13 +1,16 @@
 package com.objectcomputing.checkins.services.action_item;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
on lines 3-4, can we use javax.validation.constraints.* instead of edu.umd.cs?

Reply:
Ooh, good catch, thanks!